packages/client/hmi-client/src/components/widgets/Slider.vue

```diff
@@ -4,7 +4,10 @@
 		:style="{ width: isOpen ? contentWidth : tabWidth }"
 	>
 		<div class="slider-content-container" :style="`width: ${contentWidth}`">
-			<section class="slider-content" :style="sidePanelContentStyle">
+			<section
+				class="slider-content"
+				:style="`height: ${hasFooter ? 'calc(100% - 5rem)' : '100%'}; ${sidePanelContentStyle}`"
```

Comment:
Nitpick: Can the `100% - 5rem` thingy be incorporated into sidePanelContentStyle itself? It makes things a little easier if the logic is not scattered.

Reply:
Nope, the footer should be part of the slider, we need to add a footer slot and keep the design in here.